.github/workflows/srt.yml

```diff
@@ -52,7 +82,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ~/pnetcdf
-          key: ${{ runner.os }}-${{ env.PNETCDF_VERSION}}-pnetcdf-redo
+          key: ${{ runner.os }}-foo-v3-${{ env.PNETCDF_VERSION}}-pnetcdf-redo
```

Comment:
do we want to remove the foo-v part and the redo part?

Reply:
Not a problem, I was setting foo-v and redo while I was figuring out github workflows.

Reply:
Done